.github/workflows/reflow-version.yml

```
@@ -40,9 +40,12 @@ jobs:
 
       - name: Create installer version number
         id: version-number
+        # TODO figure out better way to handle versioning
         run: |
           python3 -m venv ../venv
           . ../venv/bin/activate
-          pip install setuptools_scm
-          echo "chia-installer-version=$(python3 ./build_scripts/installer-version.py)" >> "$GITHUB_OUTPUT"
+          python -m pip install --upgrade pip
+          pip install poetry
+          pip install "poetry-dynamic-versioning[plugin]"
```

One pip call when possible.  Well, except for the pip upgrade call before...  so...  two.

```suggestion
          pip install poetry "poetry-dynamic-versioning[plugin]"
```